### DISCUSSION TITLE
Can't reach database server

### DISCUSSION THREAD

**Discussion by Gmulti** [prisma/prisma#12508] [Q&A]
Hello,

For some time we have been having problems connecting to the database.
We are using Prisma, everything is fine for about ~12h and then we start to receive messages like "can't reach database server".
I don't understand where this is coming from as we have enough connection to our server available. We also use the instruction to get the "pgbouncer".

We also use PM2 (https://pm2.keymetrics.io/) for our different instances.
Our database is hosted on Heroku
In fact, we have to restart all instances regularly to get everything back to normal.

Do you have any idea what the problem is?

This is how we load Prisma: 
```js
import { PrismaClient } from "@prisma/client";

let prisma = new PrismaClient()

export default prisma;
```

**Comment by janpio** [comment]
> We also use the instruction to get the "pgbouncer".

Is your database actually using/behind PgBouncer?

Do you know how long it takes for the error message to show up? Any logs on the database side at Heroku?

**Comment by Gmulti** [comment]
Yes we have the pgBouncer.
It was getting faster and we could get the error between 8 and 10 hours of activity of an instance.

After some tests, the last one, doing the import of the prisma client before loading the Express server would seem to solve the problem. I would say that the connection is created at that time and is then properly used all the time rather than being created for the first time during some processing.

**Reply by pantharshit00** [reply]
Hello @Gmulti, our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

Can you also let us know if this is reproducible in the latest version?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
